src/actions/MapActions.js

```diff
@@ -57,7 +58,7 @@ export const mapHasLoaded = () => {
 export const updateRegion = (newRegion) => {
     return {
         type: MAP_REGION_CHANGE,
-        payload: newRegion
+        payload: {newRegion}
```

Comment:
Early on I had payload passing along a plain value. Increasingly throughout the app I've been having the payload pass objects with key:value pairs. Going to refactor these as I come across them to be consistent.